lib/awful/tag.lua

```diff
@@ -409,6 +409,36 @@ function tag.getgap(t)
     return tag.getproperty(t, "useless_gap") or beautiful.useless_gap or 0
 end
 
+--- Set size fill policy for the master client(s)
+-- @tparam string policy Can be set to
+-- "expand" (fill all the available workarea) or
+-- "mwfact" (fill only an area inside the master width factor)
+-- @tparam tag t The tag to modify, if null tag.selected() is used.
```

Comment:
You could write this as `@tparam[opt=tag.selected()] tag The tag to modify`.

JFI, it's OK like it is.